cpp/rte/mo_fluxes.h

```diff
@@ -35,29 +35,7 @@ class FluxesBroadband {
     int ncol = size(gpt_flux_up,1);
     int nlev = size(gpt_flux_up,2);
     int ngpt = size(gpt_flux_up,3);
-    // Check array sizes
-    //  Input arrays
-    if (size(gpt_flux_dn,1) != ncol || size(gpt_flux_dn,2) != nlev || size(gpt_flux_dn,3) != ngpt) {
-      stoprun("reduce: gpt_flux_dn array incorrectly sized");
-    }
-    if (allocated(gpt_flux_dn_dir)) {
-      if (size(gpt_flux_dn_dir,1) != ncol || size(gpt_flux_dn_dir,2) != nlev || size(gpt_flux_dn_dir,3) != ngpt) {
```

Comment:
Same question here about removing the checks.

Reply:
I believe `gpt_flux_dn_dir` could be padded, but I actually should check. I know below that `this->flux*` arrays have padded size and need the check removed.